docs/source/getting-started.mdx

```diff
@@ -46,7 +46,7 @@ your project's `node_modules` directory:
 
 Follow the instructions below to set up with either TypeScript or JavaScript:
 
-<ExpansionPanel title="Set up with TypeScript (Recommended)">
+<ExpansionPanel title="Set up with TypeScript (**Recommended**)">
 
 #### Set up with TypeScript
```

Comment:
I decided to add the plug at the end of this tutorial because ideally, folks would have a working server that they could generate types from _before_ they read the "Generating types" article. Does that feel good to you? I can always add another call out too if we feel like we want more emphasis!

Reply:
Yeah I know I added my comment here but I meant at the end of this article as a "what to do next" kinda thing

Reply:
I do mention it in the "Next Steps" section at the bottom of the article, does that work?

Reply:
Yep we're on the same page 👍